components/ScoreCard.vue

```diff
@@ -261,18 +261,124 @@
         </ul>
       </div>
 
+      <!-- loading experience -->
       <ul v-if="category === 'Manifest' && !manifestData && !noManifest">
         <li>
-          <span class="skeletonSpan"></span>
+          <div class="listSubDiv">
+            <span
+              class="cardIcon"
+              aria-hidden="true"
+            >
+              
+            </span>
+
+            <span>
+                Web Manifest is properly attached
+              </span>
+          </div>
+
+          <span class="subScoreSpan" >
+            <i class="fas fa-spinner"></i>
```

Comment:
@jgw96  If you also add fa-spin to the class list, it'll be animated spinner. https://fontawesome.com/how-to-use/on-the-web/styling/animating-icons.